src/views/CardStack.js

```diff
@@ -184,16 +184,17 @@ class CardStack extends React.Component<DefaultProps, Props, void> {
 
   _renderHeader(props: NavigationTransitionProps, headerMode: HeaderMode): ?React.Element<*> {
     const navigation = this._getChildNavigation(props.scene);
-    const header = this.props.router.getScreenConfig(navigation, 'header');
+    const header = this.props.router.getScreenConfig(navigation, 'header') || {};
 
     let HeaderComponent = Header;
-    if (header && header.bar) {
+    if (header.bar) {
```

Comment:
Is the option to change the header component called `header.bar`? It doesn't match what we've in the type definitions https://github.com/react-community/react-navigation/blob/master/src/TypeDefinition.js#L244